src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEParameterSymbol.cs

```diff
@@ -120,6 +117,19 @@ public bool TryGetWellKnownAttribute(WellKnownAttributeFlags flag, out bool valu
                 value = (theBits & ((int)flag << WellKnownAttributeDataOffset)) != 0;
                 return (theBits & ((int)flag << WellKnownAttributeCompletionFlagOffset)) != 0;
             }
+
+            public bool SetFlowAnalysisAnnotations(FlowAnalysisAnnotations value)
+            {
+                int bitsToSet = (1 << FlowAnalysisAnnotationsCompletionBit) | (((int)value & FlowAnalysisAnnotationsMask) << FlowAnalysisAnnotationsOffset);
```

Comment:
>(1 << FlowAnalysisAnnotationsCompletionBit) [](start = 32, length = 43)

This doesn't look right, it looks like ```FlowAnalysisAnnotationsCompletionBit``` is not an offset, but rather a value with the right bit set.